app/src/main/java/org/fundacionparaguaya/adviserplatform/data/repositories/SnapshotRepository.java

```diff
@@ -278,32 +278,23 @@ private boolean pullSnapshots() {
             families = familyRepository.getFamiliesNow();
         }
         setRecordsCount(families.size() * surveysNow.size());
-        for (Family family : families) {
-            for (Survey survey : surveysNow) {
 
-                if(shouldAbortSync()) return false;
-
-                //TODO Sodep: time complexity n^2
-                Log.d(TAG, stopWatch.lap(String.format("Family: %s, Survey: %s", family.getName(),
-                        survey.getDescription())));
-                success &= pullSnapshots(family, survey);
-                if(success && getDashActivity() != null) {
```

Comment:
Restore this `DashActivity` progress indicator maintaining coherence between `recordsCount` from `BaseRepository` and local `loopCount`

Reply:
I will restore `DashActivity` as requested 👌🏻